.github/workflows/pull.yml

```diff
@@ -5,6 +5,7 @@ on:
     types:
       - opened
       - synchronize
+      - ready_for_review
```

Comment:
should not need - synchronize (last push) will have run the workflow already
in fact, remove all types .. [per first note in docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request), the default is already opened/synchronize,reopened

Reply:
I will give it a try.  in my early testing the workflow was not triggered when PR is moved from `draft` -> `ready for review`.

Reply:
@algreasley based on my latest tests if we are not explicitly setting activity type `ready_for_review` then we are not covered for the scenario where we push the latest as a Draft, then move PR to Ready for review as nothing trigger the `synchronize` activity.

Reply:
I kept the PR without the activity types for further testing together, I will add them if we conclude that they are required

Reply:
ahh, I see .. as the usual "synchronize" will be bypassed by the "PR draft exclusion" ...

Reply:
so go with default, as my initial comment, and add the `ready_for_review` .. my mistake, sorry

Reply:
I explicitly added the defaults one too as per [doc](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-activity-types)